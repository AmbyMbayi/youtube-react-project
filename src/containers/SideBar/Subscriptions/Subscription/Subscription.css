.subscription__details {
    display: flex;
    align-items: center;
    padding-left: 10px;
    padding-right: 25px;
    margin-top: 25px;
}

::-webkit-scrollbar {
    width: 10px;
}

::-webkit-scrollbar-track {}

::-webkit-scrollbar-thumb {
    background-color: rgb(1, 151, 151);
    border-radius: 10px;
}

.subscription__details img {
    border-radius: 50%;
}

.subscription__details label {
    color: white;
    margin-left: 20px;
}

.broadcast__icon {
    margin-left: auto;
    font-size: 20px;
    color: rgb(63, 253, 253);
}