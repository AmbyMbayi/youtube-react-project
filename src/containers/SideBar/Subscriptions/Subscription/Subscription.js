import React from 'react';
import './Subscription.css';

function Subscription(props) {
	return (
		<div>
			<div className="subscription__details">
				<img src="http://via.placeholder.com/28x28" alt="" />
				<label>{props.label}</label>
				<i class="bx bx-broadcast broadcast__icon" />
			</div>
		</div>
	);
}

export default Subscription;
