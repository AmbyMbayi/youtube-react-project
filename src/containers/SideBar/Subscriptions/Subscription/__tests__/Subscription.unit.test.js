import React from 'react';
import { shallow } from 'enzyme';
import Subscription from '../Subscription';

describe('Subscription', () => {
	test('renders empty Subscription', () => {
		const wrapper = shallow(<Subscription />);
		expect(wrapper).toMatchSnapshot();
	});
	test('renders Subscription with labels ', () => {
		const wrapper = shallow(<Subscription label="subscriptions" />);
		expect(wrapper).toMatchSnapshot();
	});
});
