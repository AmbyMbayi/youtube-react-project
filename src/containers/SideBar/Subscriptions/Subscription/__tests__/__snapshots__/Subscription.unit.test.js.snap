// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subscription renders Subscription  1`] = `
<div>
  <div
    className="subscription__details"
  >
    <img
      alt=""
      src="http://via.placeholder.com/28x28"
    />
    <label>
      subscriptions
    </label>
    <i
      class="bx bx-broadcast broadcast__icon"
    />
  </div>
</div>
`;

exports[`Subscription renders Subscription with labels  1`] = `
<div>
  <div
    className="subscription__details"
  >
    <img
      alt=""
      src="http://via.placeholder.com/28x28"
    />
    <label>
      subscriptions
    </label>
    <i
      class="bx bx-broadcast broadcast__icon"
    />
  </div>
</div>
`;

exports[`Subscription renders empty Subscription 1`] = `
<div>
  <div
    className="subscription__details"
  >
    <img
      alt=""
      src="http://via.placeholder.com/28x28"
    />
    <label />
    <i
      class="bx bx-broadcast broadcast__icon"
    />
  </div>
</div>
`;
